CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,13 @@
+# Contributor Covenant Code of Conduct
+
+##Regulations
+
+The dry-python conference was created as a place where participants can discuss development and everything related to it with each other. All people are different, and in the heat of discussion it is easy to upset another participant with a word or deed. To make the conference comfortable for attendees, we all agree to abide by the following rules:
+
+* We do not give subjective assessments to people and do not discriminate against them
+* We do not discuss politics
+* We do not discuss religion
+* We do not discuss sex and relationships
+* We do not discuss money
+
+By participating in the conference, you agree to abide by these rules and leave the conference at the request of the organizers in case of violation. As organizers, we understand the complexity of following such rules (otherwise there would not have been so many scandals around the world) and we promise that we will seriously and carefully consider all conflict cases. If you see a violation of these rules at the conference or a mistake by the organizers in their application, please write to me in any messenger at mail@sobolevn.me.
```

Comment:
```suggestion
By participating in the community, you agree to abide by these rules and leave the community at the request of the organizers in case of violation. As organizers, we understand the complexity of following such rules (otherwise there would not have been so many scandals around the world) and we promise that we will seriously and carefully consider all conflict cases. If you see a violation of these rules at the conference or a mistake by the organizers in their application, please write to me in any messenger at [mail@sobolevn.me](mailto:mail@sobolevn.me).
```